tests/test_session.py

```diff
@@ -69,8 +69,7 @@ async def test_concurrent_session(
     assert resp.status == 200
     for token in ["test_case", "test_case2"]:
         resp = await agent.get("/test/session/traces", params={"test_session_token": token})
-        assert resp.status == 200
-        assert await resp.json() == []
+        assert resp.status == 404
 
 
```

Comment:
Can we add some regression tests to this file to cover the new functionality?

Reply:
done